CHANGELOG.md

```diff
@@ -1,27 +1,50 @@
 # Changelog
 
-## Unreleased
+## 0.2.0 - 2023-07-19
 
 ### BREAKING
 
 - **make artifact autodetect configurable - [Gankra]/[pr527]**
 
+  We now provide a new boolean key, `components.artifacts.auto`, that lets you explicitly
+  enable autodetection of artifacts. Previously, we would only enable this if you either
+  turned on `components.artifacts.cargo_dist`, or if you provided some package manager entries.
+  Since oranda _does_ also support gleaning artifacts even without `cargo-dist` support enabled,
+  we added this extra switch to let you toggle it without having to mess around with package managers.
+
+  This is a **breaking change**, as enabling `cargo-dist` support or specifying package managers does
```

Comment:
for breaking changes i think it's helpful to have really *clear* update instructions. so while i think all the information is here in this section, we may want to say something like "if you were previously relying on this autodetection and update to this version, you'll see that your artifacts are no longer displayed on your oranda page. to fix this, create an `oranda.json` if you don't already have on and add this to it:

```json
{
   "artifacts": {
       "auto": "true"
  }
}
```